src/cmap/auth/scram.ts

```diff
@@ -363,14 +337,6 @@ function compareDigest(lhs: Buffer, rhs: Uint8Array) {
   return result === 0;
 }
 
-function resolveError(err?: AnyError, result?: Document) {
```

Comment:
why is this logic removed?

Reply:
https://github.com/mongodb/node-mongodb-native/blob/NODE-5136-misc-refactors/src/cmap/connection.ts#L410

The command fn throws if there is a server error, the result condition would never be used